libs/utils/decorate.js

```diff
@@ -1,4 +1,6 @@
-import { createTag } from './utils.js';
+import { createTag, createIntersectionObserver } from './utils.js';
+
+const ROOT_MARGIN = 1000;
```

Comment:
I am thinking whether we should have this constant here or in the `turnAnchorIntoVideo` method.

Reply:
Moved into the fn directly